auth-web/src/components/auth/manage-business/AffiliatedEntityTable.vue

```diff
@@ -225,18 +224,14 @@ export default defineComponent({
       return NrState.DRAFT === state.toUpperCase()
     }
 
-    const isIA = (type: string): boolean => {
-      return (type === AffiliationTypes.INCORPORATION_APPLICATION || type === AffiliationTypes.REGISTRATION)
-    }
-
```

Comment:
Unused, removed

Reply:
On line 228, you can write that `type` is "AffiliationTypes".